lib/spells/TargetCondition.cpp

```diff
@@ -57,11 +57,7 @@ class TargetConditionItemBase : public TargetConditionItem
 	bool isReceptive(const Mechanics * m, const battle::Unit * target) const override
 	{
 		bool result = check(m, target);
-
-		if(inverted)
-			return !result;
-		else
-			return result;
+		return inverted == !result;
```

Comment:
IMO code gets less readable in cases such as this one

Reply:
Agreed. inverted probably means that result should be inverted. This case inverted ^ result should work?

Reply:
Changed for better readebility `return inverted != result; `.